### ISSUE TITLE
Rule to force position of else statement in multiline guard statements

### ISSUE THREAD

**Issue by mgrebenets** [nicklockwood/SwiftFormat#206]
Would it be possible to have a rule that enforces `else` position in multi-line `guard` statements?
E.g. 

```swift
guard
  let a = a,
  let b = b else {
  return
}
```

would be formatted into

```swift
guard
  let a = a,
  let b = b
else {
  return
}
```

At the moment SwiftFormat leaves code like this untouched.

I believe this is related to #37.

**Comment by mgrebenets** [comment]
Another useful formatting rule would be to enforce uniform formatting for multiline guards.

For example, when `guard` is multiline, but there is no newline after `guard` itself:

```swift
guard let a = a,
  let b = b
else {
 return
}
```

Could be formatted into

```swift
guard
  let a = a,
  let b = b
else {
  return
}
```

Since it's clear that `guard` is multiline here.

**Comment by nicklockwood** [comment]
@mgrebenets do you think it would make sense to tie this to the existing `elseOnSameLine` rule configuration? It seems related.

**Comment by rafcabezas** [comment]
@nicklockwood Please don't tie it into the `elseOnSameLine` rule. We use else on same line for everything except multiline guards

**Comment by rafcabezas** [comment]
A rule to enforce the multiline guard formatting would be greatly appreciated, though. With our current setup, swiftformat seems to respect/infer the current multiline guard format and leave it as is; but the reason we're using swiftformat is to enforce a certain formatting style, so it kind of defeats the purpose :)

**Comment by hartbit** [comment]
Same here.

**Comment by nicklockwood** [comment]
@mgrebenets @rafcabezas @hartbit this is now implemented in 0.45.1. It's controlled by a new `--guardelse` option.
